documentation/modules/proc-iot-api.adoc

```diff
@@ -68,4 +72,9 @@ mvn spring-boot:run -Drun.arguments=--hono.client.host=$MESSAGING_HOST,--hono.cl
 curl --insecure -X POST -i -u sensor1@myapp.iot:hono-secret -H 'Content-Type: application/json' --data-binary '{"temp": 5}' https://{http-adapter}/{api}
 ----
 
-// TODO cover using MQTT here
+. Send {api} message using MQTT:
```

Comment:
```suggestion
. Send an {api} message using MQTT:
```

Reply:
Removed {api} from the text, it will remain just "a message"